projects/ngx-resource/core/tsconfig.lib.json

```diff
@@ -12,7 +12,7 @@
     ]
   },
   "angularCompilerOptions": {
-    "annotateForClosureCompiler": true,
+    "enableIvy": false,
```

Comment:
Libraries should maintain backwards compatibility and opt out from Ivy compiler: https://angular.io/guide/ivy#maintaining-library-compatibility